packages/app/obojobo-repository/shared/components/notification.scss

```diff
@@ -0,0 +1,57 @@
+@import '~styles/includes';
+
+$poppin-pink: #af1b5c;
+$focus-pink: #fbdae6;
```

Comment:
Same as above - if this color is used in more than one place, move it to a location accessible in all the places it is used.

Also name it more descriptively, e.g. `$color-notification-focus`.